src/Core/JVM/Field/PHPDynamicField.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace PHPJava\Core\JVM\Field;
+
+use PHPJava\Core\JVM\ClassInvokerInterface;
+use PHPJava\Core\JVM\PHPClassInvoker;
+use PHPJava\Packages\java\lang\NoSuchFieldException;
+
+class PHPDynamicField implements FieldInterface
+{
+    /**
+     * @var ClassInvokerInterface|PHPClassInvoker
+     */
+    private $javaClassInvoker;
+
+    /**
+     * @var mixed[]
```

Comment:
Isn't this actually `\ReflectionProperty[]`?

Reply:
No, $fields can be set various types.
(e.g. As such as _Object, _String, JavaClass and so on)

Reply:
Then, how is it possible to call `getValue()` method?

Reply:
I'm recommending that you cast type with string on PHP.
In other way, you can  use Extractor::getRealValue method or use `instanceof` operator.

Reply:
No. I'm talking about the implementation of the methods `get(string $name)` and `set(string $name, $value)` which require the property contain `\ReflectionField` for its element.

Reply:
Sorry for misunderstanding. Your review is correct.
I fixed it.

Reply:
Yeah, that's OK 😺